tools/pipelines/test-perf-benchmarks.yml

```
@@ -546,7 +546,7 @@ stages:
             FLUID_ENDPOINTNAME: "local"
 
         - task: Bash@3
-          displayName: Remove Output Folders
+          displayName: Remove Output Folders before ODSP run
```

```suggestion
          displayName: Remove Output Folders from local server run
```
For symmetry with the new step. Seems better if it explicitly says _what_ it's trying to clean up.